### PR TITLE
APIGOV-SB001 - update based on sdk change

### PR DESCRIPTION
**Pull request by sbolosan**
update tests based on sdk change for crds

### CODE REVIEW

File: discovery/pkg/apigee/pollproxiesjob_test.go [hunk 1]
```diff
@@ -119,11 +120,13 @@ func Test_pollProxiesJob(t *testing.T) {
 				crds := []string{}
 				if tc.hasAPIKey {
 					crds = append(crds, provisioning.APIKeyCRD)
+					allowedOAuthMethods = append(allowedOAuthMethods, provisioning.APIKey)
```
Comment (jcollins-axway) [inline discovery/pkg/apigee/pollproxiesjob_test.go:123]: should not be necessary

Comment (jcollins-axway) [reply discovery/pkg/apigee/pollproxiesjob_test.go:123]: allowedCRDs?

Comment (jcollins-axway) [reply discovery/pkg/apigee/pollproxiesjob_test.go:123]: I guess the better fix for this is to drop the var and send in an empty array to the GetCred...method.

Comment (sbolosan) [reply discovery/pkg/apigee/pollproxiesjob_test.go]: done...

**Review by jcollins-axway — APPROVED** [review]

**Pull request merged**